src/tools/run-make-support/src/rustc.rs

```diff
@@ -190,6 +190,16 @@ impl Rustc {
         self
     }
 
+    pub fn extra_filename(&mut self, extra: &str) -> &mut Self {
+        self.cmd.arg(format!("-Cextra-filename={extra}"));
+        self
+    }
+
+    pub fn prefer_dynamic(&mut self) -> &mut Self {
+        self.cmd.arg("-Cprefer-dynamic");
+        self
+    }
```

Comment:
Suggestion: probably not this helper, because it doesn't really add anything.

Reply:
:+1: